### PR TITLE
Update Russian.properties

### PR DESCRIPTION
**Pull request by YOlegY**
Previous is patch-1 (1-141 lines)
141-1617 Fixed errors and added translations.
1417 Untranslateble thing .

### CODE REVIEW

File: android/assets/jsons/translationsByLanguage/Russian.properties [hunk 1]
```diff
@@ -462,27 +462,27 @@ Pay only one third the usual cost for naval unit maintenance. Melee naval units
...
-Hiawatha = Хайавата
+Hiawatha = Гайавата
 All units move through Forest and Jungle Tiles in friendly territory as if they have roads. These tiles can be used to establish City Connections upon researching the Wheel. = Все юниты двигаются по клеткам с джунглями и лесами как если бы на них были дороги. С помощью таких клеток можно установить соединение между городами после исследования Колеса.
 
 Persia = Персия
 Darius I = Дарий I
 Golden Ages last 50% longer. During a Golden Age, units receive +1 Movement and +10% Strength = Золотой век длится на 50% дольше. Во время Золотого века юниты получают +1 к движению и +10% к силе.
 
-Polynesia = Полинезия
+Polynesia = Полинезия
```
Comment (naivekook) [inline android/assets/jsons/translationsByLanguage/Russian.properties]: odd space

File: android/assets/jsons/translationsByLanguage/Russian.properties [hunk 2]
```diff
@@ -513,7 +513,7 @@ Domination = Завоевание
 Cultural = Культурная
 
 HIGHLY EXPERIMENTAL - YOU HAVE BEEN WARNED! = В РАЗРАБОТКЕ - ВЫ ПРЕДУПРЕЖДЕНЫ!
-Online Multiplayer = Online мультиплеер
+Online Multiplayer = Онлайн Мультиплеер
```
Comment (naivekook) [inline android/assets/jsons/translationsByLanguage/Russian.properties]: "Игра по сети" ?

Comment (YOlegY) [reply android/assets/jsons/translationsByLanguage/Russian.properties]: Then better "Сетевая игра"

Comment (naivekook) [reply android/assets/jsons/translationsByLanguage/Russian.properties]: agree with "Сетевая игра" :)

File: android/assets/jsons/translationsByLanguage/Russian.properties (hunk 1 above)
Comment (YOlegY) [reply android/assets/jsons/translationsByLanguage/Russian.properties]: fixed

File: android/assets/jsons/translationsByLanguage/Russian.properties (hunk 2 above)
Comment (YOlegY) [reply android/assets/jsons/translationsByLanguage/Russian.properties]: Changed

**Pull request merged**